test/test_user.py

```diff
@@ -97,3 +96,31 @@ def test_patch_username_password_length_requirements(self, helper):
         user.update(username=long_username, password=long_password)
         print("  Then when I then try to login using the newly changed data, it should also be successful")
         helper.login_with_username_password(long_username, long_password)
+
+    def test_admin_list_users(self, helper):
+        print("A super admin user can list all other users")
+        admin_user = helper.admin_user()
+        users_response = admin_user.list_users()
+        assert users_response.ok
+
+        print("A regular user can't list all other users")
+        non_admin_user = helper.given_new_user(self, "non_admin")
+        with pytest.raises(AuthorizationError):
+            print("When a regular user tries to list other users it should return an error")
+            non_admin_user.list_users()
+
+    def test_admin_user_viewing_as_other_user(self, helper):
+        admin_user = helper.admin_user()
+        users = admin_user.list_users()
```

Comment:
I would definitely create a user here.   Then there should be no need to call list_users which would mean that the test_admin user test is should succeed even if the list_users api stops working.   Making it more robust.